js/View/TrackList/Faceted.js

```diff
@@ -45,7 +45,9 @@ return declare( 'ComboTrackSelector.View.TrackList.Faceted', JBrowse.View.TrackL
         var title_list = titles.join("<br>");
 
         var icon_elem = '<div></div>';
-        if( this.browser.config.combotracksel_icon !== undefined )
+        if( this.config.icon !== undefined )
```

Comment:
@abretaud, is this change needed?

From my recollection, it is already possible to control the icon via configuration, like so (for example, via `tracks.conf`):
```
	[plugins]
	ComboTrackSelector.location += ./plugins/ComboTrackSelector
	ComboTrackSelector.icon += https://gravatar.com/avatar/f5ae7bbffab5fbd56777fee2720c7ee4.png
```

Is this not functioning as expected?